src/main/webapp/css/home_style.css

```diff
@@ -0,0 +1,13 @@
+.heading {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-end;
+  align-items: center;
+  height: 45vh;
+}
+
+h1 {
+  text-align: center;
+  margin: 20px 20px 25px;
```

Comment:
I think you'll want the margin to be on the link, not the h1 - see how currently this margin is part of the link so the clickable area is pretty large.